sopel/modules/emoticons.py

```diff
@@ -8,104 +8,108 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from sopel.module import commands, example
+from sopel import module
 
 
-@commands('shrug')
-@example('.shrug', r'¯\_(ツ)_/¯')
+@module.commands('shrug')
+@module.action_commands('shrugs')
+@module.example('.shrug', r'¯\_(ツ)_/¯')
 def shrug(bot, trigger):
     bot.say('¯\\_(ツ)_/¯')
 
 
-@commands('happy')
-@example('.happy', 'ᕕ( ᐛ )ᕗ')
+@module.commands('happy')
+@module.example('.happy', 'ᕕ( ᐛ )ᕗ')
 def happy(bot, trigger):
     bot.say('ᕕ( ᐛ )ᕗ')
 
 
-@commands('tableflip', 'tflip')
-@example('.tableflip', '(╯°□°）╯︵ ┻━┻')
-@example('.tflip', '(╯°□°）╯︵ ┻━┻')
+@module.commands('tableflip', 'tflip')
+@module.action_commands('flips table')
+@module.example('.tableflip', '(╯°□°）╯︵ ┻━┻')
+@module.example('.tflip', '(╯°□°）╯︵ ┻━┻')
 def tableflip(bot, trigger):
     bot.say('(╯°□°）╯︵ ┻━┻')
 
 
-@commands('unflip')
-@example('.unflip', '┬┬ ﻿ノ( ゜-゜ノ)')
+@module.commands('unflip')
+@module.action_commands('unflips')
```

Comment:
See above regex discussion for `flips table`; just presenting the same alternatives here.

```suggestion
@module.action_commands('unflips', 'unflips the table')
```

```suggestion
@module.action_commands('unflips(?: the table)?')
```

Reply:
Note that `unflips` already matches `unflips` and `unflips <anything>`, since commands don't check arguments.

If we need to be accurate, this can be done instead:

```suggestion
@module.action_commands('unflips table', 'unflips the table')
```

Reply:
Actually, good point. Let's keep only the ones with "table", so "/me unflips" won't do it.